src/main/java/org/fairdatapoint/config/BootstrapConfig.java

```diff
@@ -55,35 +56,31 @@
 @Slf4j
 public class BootstrapConfig {
     private final ResourcePatternResolver resourceResolver = new PathMatchingResourcePatternResolver();
-    private final boolean bootstrapEnabled;
-    private final Path dbFixturesPath;
+    private final BootstrapProperties bootstrap;
 
-    public BootstrapConfig(
-            @Value("${bootstrap.enabled:false}") boolean bootstrapEnabled,
-            @Value("${bootstrap.db-fixtures-dir}") String dbFixturesDir
-    ) {
-        this.bootstrapEnabled = bootstrapEnabled;
-        this.dbFixturesPath = Path.of(dbFixturesDir);
+    public BootstrapConfig(BootstrapProperties bootstrapProperties) {
+        this.bootstrap = bootstrapProperties;
     }
 
     @Bean
     public Jackson2RepositoryPopulatorFactoryBean repositoryPopulator() {
         final Jackson2RepositoryPopulatorFactoryBean factory = new Jackson2RepositoryPopulatorFactoryBean();
-        if (bootstrapEnabled) {
+        if (this.bootstrap.isEnabled()) {
             log.info("Bootstrap repository populator enabled");
             try {
                 // collect fixture resources
-                final Path fixturesPath = dbFixturesPath.resolve("*.json");
-                final Resource[] resources = resourceResolver.getResources("file:" + fixturesPath);
+                System.out.println(this.bootstrap.getDbFixturesDirs());
```

Comment:
Use logging instead?

Reply:
Yes, actually this is a leftover from debugging, so I'll remove it.